kolibri/utils/cli.py

```diff
@@ -469,6 +244,10 @@ def start(port, zip_port, background):
     """
     Start the server on given port.
     """
+    port = OPTIONS["Deployment"]["HTTP_PORT"] if port is None else port
+    zip_port = (
+        OPTIONS["Deployment"]["ZIP_CONTENT_PORT"] if zip_port is None else zip_port
```

Comment:
As `OPTIONS["Deployment"]["ZIP_CONTENT_PORT"] `is 0 by default, the port number is not known if it's not set by the user. I think we should add this information to https://kolibri.readthedocs.io/en/latest/access/index.html so system administrators know they need to open this port in the firewall (thinking specially of the annoying messages coming from Windows on different ports everytime kolibri starts)

Reply:
Yes, definitely - I'll add a user docs label to the PR.